src/modules/menu/components/advancedTools/MenuAdvancedToolsList.vue

```diff
@@ -1,3 +1,52 @@
+<script setup>
+import { computed, toRefs } from 'vue'
+import { useStore } from 'vuex'
+
+import ImportCatalogue from '@/modules/menu/components/advancedTools/ImportCatalogue/ImportCatalogue.vue'
+import ImportFile from '@/modules/menu/components/advancedTools/ImportFile/ImportFile.vue'
+import MenuAdvancedToolsListItem from '@/modules/menu/components/advancedTools/MenuAdvancedToolsListItem.vue'
+import ModalWithBackdrop from '@/utils/ModalWithBackdrop.vue'
+
+const props = defineProps({
+    compact: {
+        type: Boolean,
+        default: false,
+    },
+})
+
+const { compact } = toRefs(props)
+
+const store = useStore()
+
+const showImportCatalogue = computed(() => store.state.ui.importCatalogue)
+const showImportFile = computed(() => store.state.ui.importFile)
+const storeCompareRatio = computed(() => store.state.ui.compareRatio)
+const isPhoneMode = computed(() => store.getters.isPhoneMode)
+const is3dActive = computed(() => store.state.cesium.active)
+function compareSliderActive() {
+    return storeCompareRatio.value > 0.0 && storeCompareRatio.value < 1.0
+}
+
+function onToggleImportCatalogue() {
+    store.dispatch('toggleImportCatalogue')
+}
+
+function onToggleCompareSlider() {
+    // this allows us to store the previous compare ratio while making
+    // the Compare Slider invisible.
+    store.dispatch('setCompareRatio', -storeCompareRatio.value)
```

Comment:
you could give a `null` here, and in the store handle this and set it back to the default value (instead of null).
Otherwise the URL still contains the compare value, event if de-activated.

Reply:
that's a good idea